cli/src/commands/subgraph/commands/check.ts

```diff
@@ -50,8 +47,27 @@ export default (opts: BaseCommandOptions) => {
       });
     }
 
-    // submit an empty schema in case of a delete check
-    const schema = schemaFile ? await readFile(schemaFile) : Buffer.from('');
+    // Handle schema input
```

Comment:
@L42y the apollo approach in accepting the schema from stdin is unusual. Why should we use the `--schema` flag when piping data is already implicit? Additionally, `--schema` is declared as a way to specify a file path.

What do you think about this approach?

```
data | wgc subgraph check my-graph
```

This would follow the behaviour of common unix tools. An example to implement this is demonstrated [here](https://github.com/tj/commander.js/issues/137#issuecomment-50937953). This can be then also reused across all other commands.

Reply:
Let me know what you think, I think both approaches has it's benefits.